api/tests/acceptance/application/admin-members/admin-members-routes_test.js

```diff
@@ -46,8 +46,8 @@ describe('Acceptance | Application | Admin-members | Routes', function () {
       });
 
       // then
-      expect(response.statusCode).to.equal(404);
-      expect(response.statusMessage).to.equal('Not Found');
+      expect(response.statusCode).to.equal(403);
```

Comment:
Il faudrait mettre à jour le titre du test 

```
// BAD
should return 404 http status code when user is not a member of pix admin and has no role

// GOOD
should return 403 http status code when user is not a member of pix admin and has no role
```

Et peut être mettre ce test dans un `context` avec le **when** du test `when user is not a member of pix admin and has no role`